Web/src/components/games/BalloonShake/GameDisplay.tsx

```diff
@@ -1,66 +1,63 @@
 import React from 'react';
 import styled from 'styled-components';
-import { ReactComponent as BalloonSVG } from 'svg/balloon.svg';
+import TimerDisplay from 'components/games/TimerDisplay';
+import { ReactComponent as Balloon } from 'svg/balloon.svg';
 
 interface IProps {
   secondsLeft: number;
   count: number;
 }
 
 const GameContainer = styled.div`
-  background: var(--pale-yellow);
+  background: var(--pink);
   overflow: hidden;
-  height: 100vh;
-  width: 100vw;
+  height: ${window.innerHeight}px;
+
   display: flex;
-  justify-content: center;
+  justify-content: space-evenly;
   flex-direction: column;
   align-items: center;
-  z-index: -999;
+
+  color: white;
+  font-size: 1.4rem;
+  text-shadow: 3px 3px 0px rgba(0, 0, 0, 0.1);
 
   & > * {
     z-index: 1;
   }
 `;
 
-const Balloon = styled(BalloonSVG)`
+const BalloonContainer = styled.div`
   display: flex;
-  position: absolute;
   justify-content: center;
+  align-items: flex-end;
+  height: 55vh;
   z-index: 0;
 `;
 
-const TimeLeft = styled.h2`
-  font-size: 6rem;
-  color: var(--purple);
-  margin: 0 0 0 0;
-  justify-content: center;
+const ShakeCountSection = styled.section`
+  display: flex;
+  justify-content: flex-start;
+  align-items: baseline;
 
-  // Increase line-height to curb font rendering bug
-  line-height: 1.2;
```

Comment:
What’s the story with the bug?

Reply:
the font is broken and too tall to fit within its height. So on some rerenders the older pixels above the height don't disappear

Reply:
As in this got removed, so I presume this is not a problem anymore or some other CSS attributes solved it?

Reply:
LOL nah, it's "removed" here because I moved the timer styles into the `TimerDisplay` file where the timer is now in

Reply:
👍